packaging/docker/run.sh

```diff
@@ -23,7 +23,7 @@ function add_netdata_to_proxmox_conf_files_group() {
     fi
   fi
 
-  if ! getent group "${group_guid}" | grep -q netdata; then
+  if ! getent group "${group_guid}" | grep -q "^netdata:"; then
```

Comment:
This change has nothing to do with the PR and it breaks the code.

This checks if the `netdata` user is a member of the group with GID `${group_guid}`.

The output format of `getent group` is:

`group_name:password_placeholder:GID:member_list`

So `grep -q netdata` correctly checks if "netdata" appears anywhere in the line, which would indicate the `netdata` user is a member of this group.

Now you're checking if the line starts with "netdata:" which means you're checking if the group's name is "netdata", not if the user "netdata" is a member of the group.

Reply:
Oops, my bad. I thought it was checking whether the group was named netdata. Sorry!

```suggestion
  if ! getent group "${group_guid}" | grep -q netdata; then
```